simd/2022/SIMD-03-18.md

```diff
@@ -20,9 +20,134 @@ You are a new attendee, please fill out this
 1.  Find volunteers for note taking
 1.  Adoption of the agenda
 1.  Proposals and discussions
-    1.  *add agenda items here*
+    1.  Relaxed SIMD Progress updates (Zhi An Ng)
+    1.  AOB
 1.  Closure
 
 ## Meeting notes
 
 ### Attendees
+
+- Zhi An Ng
+- Yury Delendik
+- Lars Hansen
+- Andrew Brown
+- Richard Winterton
+- Thomas Lively
+- Petr Penzin
```

Comment:
Johnnie Birch joined later, I missed that.

Reply:
Thanks, updated.